src/SectorView.cpp

```diff
@@ -532,6 +549,188 @@ void SectorView::DrawNearSectors(const matrix4x4f& modelview)
 	Gui::Screen::LeaveOrtho();
 }
 
+bool SectorView::MoveRouteItemUp(const std::vector<SystemPath>::size_type element) {
+	if (element <= 0 || element >= m_route.size()) return false;
+
+	SystemPath tmp = m_route[element - 1];
```

Comment:
Would std::swap work here? http://en.cppreference.com/w/cpp/algorithm/swap

Reply:
Yes, I imagine so. This is me slipping back into my old C habits...